### PR TITLE
Update PageFetcherSnapshotTest asserts with containsExactly

### PR DESCRIPTION
**Pull request by claraf3**
## Proposed Changes

Update assertThat tests from using isEqualTo to containsExactly.

## Testing

Test: ./gradlew paging:paging-common:test

### CODE REVIEW

File: paging/common/src/test/kotlin/androidx/paging/PageFetcherSnapshotTest.kt [hunk 1]
```diff
@@ -249,15 +234,13 @@ class PageFetcherSnapshotTest {
...
-            listOf<PageEvent<Int>>(
-                LoadStateUpdate(APPEND, false, Loading),
-                createAppend(
-                    pageOffset = 1,
-                    range = 99..99,
-                    startState = NotLoading.Incomplete,
-                    endState = NotLoading.Complete
-                )
+        assertThat(fetcherState.newEvents()).containsExactly(
+            LoadStateUpdate<PageEvent<Int>>(APPEND, false, Loading),
```
Comment (dlam) [inline paging/common/src/test/kotlin/androidx/paging/PageFetcherSnapshotTest.kt]: Are the generics still needed to be explicitly declared here? If not, can we remove them also while we're doing this cleanup?

Comment (claraf3) [reply paging/common/src/test/kotlin/androidx/paging/PageFetcherSnapshotTest.kt]: If I don't provide it, "Not enough information to infer type variable T" error shows up. It appears to be required for LoadStateUpdate and Drop events.

**Comment by dlam** [comment]
Thanks for doing this cleanup - can you check if we can also remove the explicit generic declarations?

File: paging/common/src/test/kotlin/androidx/paging/PageFetcherSnapshotTest.kt [hunk 2]
```diff
@@ -110,14 +108,12 @@ class PageFetcherSnapshotTest {
...
-        assertThat(fetcherState.newEvents()).isEqualTo(
-            listOf<PageEvent<Int>>(
-                LoadStateUpdate(PREPEND, false, Loading),
-                createPrepend(
-                    pageOffset = -1,
-                    range = 0..0,
-                    startState = NotLoading.Complete
-                )
+        assertThat(fetcherState.newEvents()).containsExactly(
+            LoadStateUpdate<PageEvent<Int>>(PREPEND, false, Loading),
```
Comment (dlam) [inline paging/common/src/test/kotlin/androidx/paging/PageFetcherSnapshotTest.kt]: Shouldn't this be just `LoadStateUpdate<Int>(..)` ?

Comment (claraf3) [reply paging/common/src/test/kotlin/androidx/paging/PageFetcherSnapshotTest.kt]: You're right! Updating now.

**Comment by dlam** [comment]
Build should be fixed now - can you rebase your changes on top of latest updates on androidx-main and see if the checks pass now?

**Review by dlam — APPROVED** [review]

**Review by dlam — APPROVED** [review]

**Pull request closed without merging**